README.md

```diff
@@ -122,17 +122,15 @@ pip install apache-airflow[postgres,google]==1.10.12 \
 
-- In `master` branch of Airflow and in `v1-10-stable` branch we provide Dockerfiles and accompanying
-  files that allow to build your own customized version of the Airflow Production image. To build your own image or to customize it; first clone the image and then checkout the right version which are conservative, masters and if you are adventurous then you can run `docker build`.You can add various arguments to customize it.More instructions on how to build your own image with additional dependencies (if needed) are provided in the [IMAGES.rst](IMAGES.rst#production-images) if you want to build it using `docker build` command or in [BREEZE.rst](BREEZE.rst#building-production-images) to use Breeze tool which easier interface, auto-complete, and accompanying screencast video. Breeze is a development and text environment that is developed for airflow but it also supports building production image very easily so we specify the production image flag additional extras or python version or python dev. so most of the parameters you can specify here is in command line parameters which have auto completion option. Note, that while it is possible to use master branch to build images for released Airflow versions, it might at times get broken so you should rather rely on building your own images from the v1-10-stable branch.
+Following the ASF rules, the source packages released must be sufficient for a user to build and test the release provided they have access to the appropriate platform and tools.
 
-Airflow Summit 2020's "Production Docker Image" talk where context, architecture and customization/extension methods are [explained](https://youtu.be/wDr3Y7q2XoI).
+Other ways of retrieving source code are "convenience" methods. For example, tagging in GitHub is a convenient way to mark the git project sources that were used to generate the official source packages. We also have binary "convenience" packages how Airflow is distributed: PyPI releases which are a convenient way of installing Apache Airflow using standard python tools or Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow). Those are not official releases but they are built using officially released sources.
+
+> Note: Airflow Summit 2020's "Production Docker Image" talk where context, architecture and customization/extension methods are [explained](https://youtu.be/wDr3Y7q2XoI).
```

Comment:
```suggestion
Other ways of retrieving source code are "convenience" methods. For example, tagging in GitHub is a convenient way to mark the git project sources that were used to generate the official source packages. We also have binary "convenience" packages: PyPI releases which are a convenient way of installing Apache Airflow using standard python tools or Docker Images published in the [Apache Airflow DockerHub](https://hub.docker.com/repository/docker/apache/airflow). Those are not official releases but they are built using officially released sources.

> Note: Airflow Summit 2020's "Production Docker Image" talk where context, architecture and customization/extension methods are [explained](https://youtu.be/wDr3Y7q2XoI).
```

Reply:
LGTM otherwise. I am happy with the wording's though, making the README concise and small (if other's think so too ) can be a separate issue/PR.

Reply:
Fully Agree. The smaller and more concise README we can have, the better. It should have all the relevant links to other docs (maybe an automated checs if all the .md /.rst files are reachable from the README) and should have  only most important info. It can be indeed separate issue though (but we this one we are already decreasing the readme a bit)